recurly/__init__.py

```diff
@@ -390,6 +395,33 @@ class GiftCard(Resource):
     _classes_for_nodename = {'recipient_account': Account,'gifter_account':
             Account}
 
+    def preview(self):
+        if hasattr(self, '_url'):
+            url = self._url + '/preview'
+            return self.post(url)
+        else:
+            url = urljoin(recurly.base_uri(), self.collection_path) + '/preview'
+            return self.post(url)
+
+    def redeem(self, account_code):
+        # """Redeem this gift card on the specified account code"""
```

Comment:
Remove the `#` here for this to be a proper docstring: https://www.python.org/dev/peps/pep-0257/